roles/pigeon/templates/mainnet/config.yaml.j2

```diff
@@ -16,55 +16,55 @@ evm:
     chain-id: 1
     base-rpc-url: {{ vault.mainnet.paloma.eth_rpc }}
     keyring-pass-env-name: KEYRING_PASSWORD
-    signing-key: 0x9a27f79AE2C5342174272874C2385DCb0B8C3d07
+    signing-key: 0x0d2dbd27b81E240c6f56b2C906a4b2ffD34D3a79
     keyring-dir: {{ pigeon_keys_dir }}
     gas-adjustment: 2
     tx-type: 2
   bnb-main:
     chain-id: 56
     base-rpc-url: {{ vault.mainnet.paloma.bnb_rpc }}
     keyring-pass-env-name: KEYRING_PASSWORD
-    signing-key: 0x9a27f79AE2C5342174272874C2385DCb0B8C3d07
+    signing-key: 0x0d2dbd27b81E240c6f56b2C906a4b2ffD34D3a79
     keyring-dir: {{ pigeon_keys_dir }}
     gas-adjustment: 1
     tx-type: 0
   matic-main:
     chain-id: 137
     base-rpc-url: {{ vault.mainnet.paloma.matic_rpc }}
     keyring-pass-env-name: KEYRING_PASSWORD
-    signing-key: 0x9a27f79AE2C5342174272874C2385DCb0B8C3d07
+    signing-key: 0x0d2dbd27b81E240c6f56b2C906a4b2ffD34D3a79
     keyring-dir: {{ pigeon_keys_dir }}
     gas-adjustment: 2
     tx-type: 0
   op-main:
     chain-id: 10
     base-rpc-url: {{ vault.mainnet.paloma.op_rpc }}
     keyring-pass-env-name: KEYRING_PASSWORD
-    signing-key: 0x9a27f79AE2C5342174272874C2385DCb0B8C3d07
+    signing-key: 0x0d2dbd27b81E240c6f56b2C906a4b2ffD34D3a79
     keyring-dir: {{ pigeon_keys_dir }}
     gas-adjustment: 2
     tx-type: 2
   base-main:
     chain-id: 8453
     base-rpc-url: {{ vault.mainnet.paloma.base_rpc }}
     keyring-pass-env-name: KEYRING_PASSWORD
-    signing-key: 0x9a27f79AE2C5342174272874C2385DCb0B8C3d07
+    signing-key: 0x0d2dbd27b81E240c6f56b2C906a4b2ffD34D3a79
     keyring-dir: {{ pigeon_keys_dir }}
     gas-adjustment: 2
     tx-type: 2
   gnosis-main:
     chain-id: 100
     base-rpc-url: {{ vault.mainnet.paloma.gnosis_rpc }}
     keyring-pass-env-name: KEYRING_PASSWORD
-    signing-key: 0x9a27f79AE2C5342174272874C2385DCb0B8C3d07
+    signing-key: 0x0d2dbd27b81E240c6f56b2C906a4b2ffD34D3a79
```

Comment:
lets add a variable SIGNING_KEY in general.env similar to KEYRING_PASSWORD, would be better to update in one place